ledger/acctupdates.go

```diff
@@ -1045,14 +1060,33 @@ func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookb
 				return
 			}
 		}
+
```

Comment:
this isn't from this PR, but it looks like the return value from `updateAccountsRound` is getting ignored above if `isCatchpointRound` is true.

Reply:
That should be fixed.. I'll add this to this PR.